### DISCUSSION TITLE
It would be possible to implement the full-width address bar pop-up?

### DISCUSSION THREAD

**Discussion by Aleksandra0056** [black7375/Firefox-UI-Fix#652] [Q&A] [answered]
It would be possible to implement full-width address bar into Firefox UI Fix? Just tell me, if the JavaScript would be needed.

I think, that the full-width address bar was better (It was in a previous redesign called Photon).

Screenshot:
![Full-width address bar](https://user-images.githubusercontent.com/69351811/220373681-74269145-3261-48a9-bcc9-29925970c73b.png)

**Comment by JohnyP36** [comment]
If you move the icons right of the address bar to the overflow menu and remove the search box your address bar will automatically become full-width.

![afbeelding](https://user-images.githubusercontent.com/81161435/220386424-0143fbe8-6496-484d-89a2-f2df75c039eb.png)

**Reply by Aleksandra0056** [reply]
No, you didn't understood! I think full-width address bar pop-up. Just look at the attached screenshot.

**Reply by JohnyP36** [reply]
Aha, oké. Now I understand the problem. 
I think it can be fixed only by CSS. So, now js required.

**Reply by Aleksandra0056** [reply]
So, the JavaScript wouldn't be required?

**Reply by JohnyP36** [reply]
I don't think so. (not required)

**Reply by Aleksandra0056** [reply]
@JohnyP36 by the way, what theme is in your Firefox? It looks like, that it is Microsoft Edge-like themed.

**Reply by JohnyP36** [reply]
@Patricek0056 
> JohnyP36 by the way, what theme is in your Firefox? It looks like, that it is Microsoft Edge-like themed.

Yes, I'm using https://github.com/BPower0036/UserChrome/tree/main/FireFox (copy all files and folder to your `chrome` folder, but put the content of the `user.js` file in `C:\Users\%Username%\AppData\Roaming\Mozilla\Firefox\Profiles\x.default-release\prefs.js` (only copy rules starting with `user_pref(`.) 

BTW, that repo is based on this one.

I edited the files shown in that repo of BPower0036, so you will not get the same as I. 
If you want the exact lay-out, I can upload my files?...

**Comment by black7375** [comment, marked as answer]
It's a possible.

**Reply by Aleksandra0056** [reply]
And, are you planning to implement?

**Reply by black7375** [reply]
It may be implemented later, but it will not be the default.
It seems to cover too much web content.
